docs/evm/cadence/interacting-with-coa.md

```diff
@@ -334,19 +365,218 @@ transaction(evmContractHex: String, signature: String, args: [AnyStruct], gasLim
     }
 }
 ```
+
 :::info
-Notice that the calldata is encoded in the scope of the transaction. While developers can encode the calldata outside the scope of the transaction and pass the encoded data as an argument, doing so compromises the human-readability of Cadence transactions.
 
-It's encouraged to either define transactions for each COA call and encoded the hardcoded EVM signature and arguments, or to pass in the human-readable arguments and signature and encode the calldata within the transaction. This ensures a more interpretable and therefore transparent transaction.
+Notice that the calldata is encoded in the scope of the transaction. While developers can encode the calldata
+outside the scope of the transaction and pass the encoded data as an argument, doing so compromises the
+human-readability of Cadence transactions.
+
+It's encouraged to either define transactions for each COA call and encoded the hardcoded EVM signature and arguments,
+or to pass in the human-readable arguments and signature and encode the calldata within the transaction. This ensures a
+more interpretable and therefore transparent transaction.
+
 :::
 
-### Deploying a Contract to Flow EVM
+### Transferring FLOW in EVM
+
+Similar to transferring ETH and other native value in other EVMs, you'll want to call to the target EVM address with
+empty calldata and providing the transfer value.
+
+```cadence transfer_evm_flow.cdc
+import "EVM"
 
-To deploy a contract to the EVM, you can use the `deploy` function provided by the COA resource. This function takes the contract code, gas limit, and value you want to send. It will return the EVM address of the newly deployed contract.
+/// Transfers FLOW to another EVM address from the signer's COA
+///
+/// @param to: the serialized EVM address of the recipient
+/// @param amount: the amount of FLOW to send
+transaction(to: String, amount: UInt) {
+
+    let recipient: EVM.EVMAddress
+    let recipientPreBalance: UInt
+    let coa: auth(EVM.Call) &EVM.CadenceOwnedAccount
+    
+    prepare(signer: auth(BorrowValue) &Account) {
+        self.recipient = EVM.addressFromString(to)
+        self.recipientPreBalance = self.recipient.balance().attoflow
+        self.coa = signer.storage.borrow<auth(EVM.Call) &EVM.CadenceOwnedAccount>(from: /storage/evm)
+            ?? panic("No COA found in signer's account")
+    }
+
+    execute {
+        let res = self.coa.call(
+            to: self.recipient,
+            data: [],
+            gasLimit: 100_000,
+            value: EVM.Balance(attoflow: amount)
+        )
+
+        assert(res.status == EVM.Status.successful, message: "Failed to transfer FLOW to EVM address")
+    }
+
+    post {
+        self.recipient.balance().attoflow == self.recipientPreBalance + amount:
+            "Problem transferring value to EVM address"
```

Comment:
Same comment here about the error message. Lets describe exactly what is going wrong with numbers to show what is happening and how to fix